pkg/3scale/amp/operator/system_reconciler.go

```diff
@@ -90,7 +92,7 @@ func (r *SystemReconciler) Reconcile() (reconcile.Result, error) {
 	}
 
 	// System CM
-	err = r.ReconcileConfigMap(system.SystemConfigMap(), reconcilers.CreateOnlyMutator)
+	err = r.ReconcileConfigMap(system.SystemConfigMap(), systemConfigMapMutator)
```

Comment:
so this was a create only mutator. Meaning we are allowing changing other values post create.
Is that logic now changes and we will overwrite the operator expected values with potentially updated values?

Reply:
Yes good catch, any user-defined values in the `zync.yml` field would be overwritten. I'll tweak the mutator to account for that

Reply:
Fixed in latest commit